### DISCUSSION TITLE
Insert parent node

### DISCUSSION THREAD

**Discussion by antanasb** [nadrad/h-m-m#70] [Ideas]
User of FreeMind/FreePlane for many years. Just discovered the h-m-m and like it very much. I miss the possibility to insert parent node (of course workaround exists).

**Comment by nadrad** [comment]
I'm glad you like it.

How does it work? Does it add a new node between the selected one and its former parent?
What's a typical use case for it?

**Comment by antanasb** [comment]
Hi Nader, you're absolutely right. IMHO it's very useful feature, because for example building knowledge tree often begins from leafs by adding grouping or abstraction layers  afterwards.
